kotlin-android/app/src/main/java/io/realm/examples/objectserver/model/SwipeToDeleteCallback.kt

```diff
@@ -0,0 +1,61 @@
+package io.realm.examples.objectserver.model
+
+import android.content.Context
+import android.graphics.*
+import android.graphics.drawable.ColorDrawable
+import androidx.core.content.ContextCompat
+import androidx.recyclerview.widget.ItemTouchHelper
+import androidx.recyclerview.widget.RecyclerView
+import io.realm.examples.objectserver.R
+
+
+abstract class SwipeToDeleteCallback(context: Context) : ItemTouchHelper.SimpleCallback(0, ItemTouchHelper.LEFT) {
```

Comment:
You borrowed this from kitek and need to credit. Or don't use Swipe to Delete if it's not idiomatic to Android.

Reply:
I don't think this should be part of the tutorial.